### ISSUE TITLE
Funky layout on donations page makes it tricky to donate

### ISSUE THREAD

**Issue by lippytak** [codeforamerica/codeforamerica.org#1523]
![screen shot 2015-12-24 at 4 34 29 pm](https://cloud.githubusercontent.com/assets/2533112/11998625/4b2124e0-aa5c-11e5-9899-16c48e937606.png)

**Comment by tvanantwerp** [comment]
Agreed. I just had the same problem--had to stop for a minute and figure out, OK, which one of these radio buttons is the Visa button?

![image](https://cloud.githubusercontent.com/assets/2487968/12199381/b2122f98-b5e5-11e5-9621-b023dce44e4c.png)

Listing them horizontally like the amount radio buttons would be much clearer.

![image](https://cloud.githubusercontent.com/assets/2487968/12199392/c9bee118-b5e5-11e5-992f-950c4f3b306c.png)
